libshaderc_util/src/compiler_test.cc

```diff
@@ -92,25 +92,36 @@ class DummyCountingIncluder : public shaderc_util::CountingIncluder {
 // A test fixture for compiling GLSL shaders.
 class CompilerTest : public testing::Test {
  public:
-  // Returns true if the given compiler successfully compiles the given shader
-  // source for the given shader stage.  No includes are permitted, and shader
-  // stage deduction falls back to an invalid shader stage.
-  bool SimpleCompilationSucceeds(std::string source, EShLanguage stage) {
+  // Returns a tuple consists of two fields. 1) a boolean whose value is true if
+  // the compilation of the given shader source for the given shader stage
+  // succeeded, false otherwise; 2) the compilation output data in a vector of
+  // 4-byte words.
+  // No includes are permitted, and shader stage deduction falls back to an
+  // invalid shader stage.
+  std::tuple<bool, std::vector<uint32_t> > SimpleCompilationSucceeds(
```

Comment:
There was no reason to change the signature of SimpleCompilationSucceeds.  You only ever want the boolean answer.  In the changes below, you only ever use std::ignore for the second part of the result.